docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,215 @@
+
+**Travis:** The Travis build status and results are linked near the bottom of your PR in GitHub. 
+You can restart Travis if it gives error messages that seem inappropriate, or if the build stalls (for longer than 1h).
+if you are pushing a last typo fix on a PR, it is okay to add `[skip ci]` in the commit message. It is not okay to do this if it's the first commit or if you are fixing more than documentation typos.
+
+**Documentation PRs:** If working specifically on markdown files or other documentation, be sure to check the actual look of the document once you have pushed your PR and fix any formatting errors. You can do this by going to “Files Changed” and viewing how your markdown file will actually appear in GitHub. Don’t forget to click all links to make sure they won't be broken when the document is rendered. 
+
+## 4. Code Review
+
+### Testing or Contributing to someone's PR
+First, make a remote of their fork:
+
+    git remote add "their_fork" <their_fork_url>
+          
+Then, fetch their remote fork and checkout the branch related to their PR:
+
+    git fetch their_fork
+    git checkout their_fork/branch_name
+
+Alternatively, to avoid being under the detached HEAD state, checkout and create a local branch with the command
+
+    git checkout -b branch their_fork/branch_name
+
+Follow the provided testing instructions to test their code on your VM. 
+
+If you made changes (with their permission), commit and push them to that branch:
+
+    git commit -m "Changes made"
+    git push their_fork branch_name
+
+Don’t forget to consult the Links (see above) for coding review standards and guidelines and the code review checklist.
+
+### Labels
+The `Passed Manual Tests` label can be added to a PR if a reviewer has tested the code on their own VM and determined that it performs as intended.
+
+The `Needs Rebase` label can be added if a PR has merge conflicts with the branch it is based off of.
+
+The `Needs Work` label can be added if a PR is not yet ready for review and requires more work before it can be approved.
+
+Other Labels can also be added by reviewers at their discretion. 
+
+## 5. Rebasing
+Rebasing is used when there are conflicts between your branch and the branch your PR is based off of (an alternative to merging). 
+Here is a general workflow:
+
+    cd /var/www/loris
+    git checkout branch_name
+    git fetch aces
+    git rebase aces/main
+    (#) while (conflicts)  { 
+        // fix any conflicts
+        git add  filename
+        git rebase --continue
+    # } loop back to (#) - fix any remaining conflicts.... until no more conflicts
+    git push --force my_fork branch_name
+    # --force is often needed here :\
+
+### Tips for fixing a rebase
+1. The most basic way of “fixing” a rebase is to go back to the commit right before the rebase. The easiest way to do this is to either reset or revert to a previous commit. 
```

Comment:
Why do we need to fix a rebase? I'm not sure I understand this section :)

Reply:
Sometimes rebases get messed up. I can change the title to something like "How to fix a rebase gone wrong" or something

Reply:
What is suggested here is a destructive operation. I suggest to perform a backup first. And also not sure why would want to destroy commits after a rebase?